File: csharp/examples/HelloWorld.cs
using DriftGateSdk;

var client = new DriftGateClient("https://api.driftgate.ai", bearerToken: "token");
var session = await client.Session.StartAsync(new SessionStartRequest("refund-agent"));
await session.ExecuteAsync(new ExecutionRequest(new Dictionary<string, object?> { ["orderId"] = "123" }));

File: php/examples/hello_world.php
<?php

declare(strict_types=1);

require_once __DIR__ . '/../src/DriftGateClient.php';

use DriftGate\Sdk\DriftGateClient;

$client = new DriftGateClient('https://api.driftgate.ai', bearerToken: 'token');
$session = $client->sessionStart(agent: 'refund-agent');
$session->execute(input: ['orderId' => '123']);
